src/components/global/footer.tsx

```diff
@@ -1,9 +1,223 @@
+  width: 44px;
+  height: 44px;
+  margin: 0.7rem;
+  padding: 0;
+  border: 2px solid ${colors.blackLight};
+  transition: 0.3s ease;
+  border-radius: 0.75rem;
+  vertical-align: middle;
+
+  &:hover {
+    background-color: ${colors.accent};
+    border-color: ${colors.accent};
+  }
+`;
+
+const StyledLinkWrapper = styled.div`
+  display: flex;
+  height: 9px;
+  color: #F8F8F8;
+  font-weight: 400;
+  opacity: 0.5;
+  margin-bottom: 1.5rem;
+  justify-content: end;
+
+  & > a {
+    margin-right: 1.5rem;
+  }
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    justify-content: center;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    justify-content: center;
+  }
+`;
+
+const StyledSlider = styled.div`
+  height: 155px;
+  margin: auto;
+  width: 100%;
+  display: grid;
+  place-items: center;
+  overflow: hidden;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    height: 120px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    height: 70px;
+  }
+`;
+
+const StyledSlideTrack = styled.div`
+  display: flex;
+  width: calc(1725px * 4); // total width of all slides
+  animation: scroll 8s linear infinite;
+
+  @keyframes scroll {
+    0% {
+      transform: translateX(0);
+    }
+    100% {
+      transform: translateX(-1725px);
+    }
+  }
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    width: calc(1200px * 4);
+    animation: scroll 6s linear infinite;
+
+    @keyframes scroll {
+      0% {
+        transform: translateX(0);
+      }
+      100% {
+        transform: translateX(-1200px);
+      }
+    }
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    width: calc(800px * 4);
+    animation: scroll 4s linear infinite;
+
+    @keyframes scroll {
+      0% {
+        transform: translateX(0);
+      }
+      100% {
+        transform: translateX(-600px);
+      }
+    }
+  }
+`;
+
+const StyledSlide = styled.div`
+  width: 1725px;
+  display: flex;
+  align-items: start;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    width: 1000px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    width: 600px;
+  }
+`;
+
+const StyledSlideText = styled.span`
+  font-weight: 600;
+  font-size: 344px;
+  letter-spacing: 0.6rem;
+  width: 100%;
+  line-height: 72px;
+  text-shadow: 9px 0px #39979C;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    font-size: 185px;
+    line-height: 40px;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    font-size: 103px;
+    line-height: 21.6px;
+    text-shadow: none;
+  }
+`;
 
 export default function Footer() {
   return (
     <footer>
-      <h2>Footer will go here</h2>
+      <StyledWrapper>
+        <StyledContent>
+          <StyledParagraphWrapper>
+            <StyledTextSpacer>{"Interested?"}</StyledTextSpacer>
+            <StyledPillWrapper>
+              <Pill pillText={"Interested?"} />
+            </StyledPillWrapper>
+            <StyledParagraphText>
```

Comment:
To me, this looks smaller than the Our Approach text, I see you are using the styles from that page so I'm not quite sure why. But be aware the `StyledParagraphText` element is a `span` and not a `p`, but if you can take second look and tell me if you also think it's a different size

Reply:
Same on mobile

Reply:
I had the same impression, but they have the same size. I think the gray background makes the text pop up more in Our Approach, so I removed it when testing and it looked pretty much the same as in the Footer

Reply:
Yeah you're right seems to match when I put them side by side